spec/GameSpec.js

```diff
@@ -0,0 +1,216 @@
+describe ('Game', function() {
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it ('starts with 0 scores', function() {
+    expect(game.total).toBe(0);
+  });
+
+  it ('has internally 12 frames', function() {
+    expect(game.frames.length).toBe(12);
+  });
+
+  it ('has 2 rolls in each frame', function() {
+    expect(game.frames[0].length).toBe(2);
+  });
+
+  it ('stores the number of fallen pins in each roll', function() {
+    game.setScores(1, 1, 7);
+    expect(game.frames[0][0]).toBe(7);
```

Comment:
This is breaking encapsulation as you're directly accessing the frames array from 'outside' the object.
Consider:
```js
game.getRollScore(1, 1)
```